pandas/tests/arrays/sparse/test_accessor.py

```diff
@@ -119,3 +119,11 @@ def test_series_from_coo(self, dtype, dense_index):
             )
 
         tm.assert_series_equal(result, expected)
+
+    @td.skip_if_no_scipy
+    def test_series_from_coo_incorrect_format(self):
+        # non-regression test for fix to issue #26554
+        import scipy.sparse
+        m = scipy.sparse.csr_matrix(np.array([[0, 1], [0, 0]]))
+        with pytest.raises(TypeError):
```

Comment:
can you add a message check

Reply:
Done!

Reply:
as this test is not parametrised can you just use the literal message raised by this test instead.

Reply:
Will do!